qcfractal/storage_sockets/sqlalchemy_socket.py

```diff
@@ -1817,8 +1864,10 @@ def _copy_managers(self, record_list: Dict):
 
                 if get_count_fast(doc) == 0:
                     doc = QueueManagerORM(**manager)
-                    doc.created_on = dt.fromtimestamp(doc.created_on / 1e3)
-                    doc.modified_on = dt.fromtimestamp(doc.modified_on / 1e3)
+                    if isinstance(doc.created_on, float):
+                        doc.created_on = datetime.datetime.fromtimestamp(doc.created_on / 1e3)
```

Comment:
It looks like this is a incorrect merge as well. Can you please change the signature to `dt.fromtimestamp` and remove the `import datetime` at the top?

Reply:
I added the check `isinstance`, but maybe I was using the `datetime` as it was in the file.

Reply:
Once this and the env issue is fixed, I will go ahead and merge.